tensorboard/data/server/cli.rs

```diff
@@ -114,6 +116,17 @@ struct Opts {
     )]
     #[allow(unused)]
     no_checksum: bool,
+
+    /// Set explicit series sampling
+    ///
+    /// A comma separated list of plugin_name=num_samples pairs to explicitly specify how many
```

Comment:
nit: Rustdoc is Markdown; backticks around `` `plugin_name=num_samples` ``
and around `` `num_samples` `` below?